.github/workflows/static.yml

```diff
@@ -11,30 +11,20 @@ jobs:
     name: static
     runs-on: ubuntu-latest
     steps:
-      - name: Check GitHub rate limit at start
-        uses: wakamex/rate-limit-action@master
-        with:
-          token: ${{ secrets.GH_GITHUB_COM_TOKEN || github.token}}
-
       - uses: actions/checkout@v3
         with:
-          token: ${{ secrets.GH_GITHUB_COM_TOKEN || github.token}}
+          token: ${{github.token}}
 
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
           python-version: "3.10"
           cache: "pip"
-          token: ${{ secrets.GH_GITHUB_COM_TOKEN || github.token}}
+          token: ${{github.token}}
       - run: |
           python -m pip install --upgrade pip
           python -m pip install -r requirements.txt
           python -m pip install -r requirements-dev.txt
 
       - name: Analysing code with pyright
-        run: python -m pyright $(git ls-files '*.py' '*.pyi')
-
-      - name: Check GitHub rate limit at end
-        uses: wakamex/rate-limit-action@master
-        with:
-          token: ${{ secrets.GH_GITHUB_COM_TOKEN || github.token}}
+        run: python -m pyright $(git ls-files '*.py' '*.pyi')
```

Comment:
newline went away -- do we need it in the yamls?

Reply:
CI ran fine, so I guess not